xml/adm_sudo.xml

```diff
@@ -0,0 +1,611 @@
+              If a command is given, it is passed to the shell (with the <option>-c</option> option),
+              else the shell is run in interactive mode.
+            </para>
+            <screen>
+<prompt>&exampleuser_plain;:~ &gt; </prompt>sudo -i
+root's password:
+<prompt>root:/home/tux # </prompt>exit
+<prompt>&exampleuser_plain;:~ &gt; </prompt>
+</screen>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term>
+            <literal>sudo -i (&lt;command&gt;)</literal>
+          </term>
+          <listitem>
+            <para>
+              Like <option>-s</option>, but start the shell as login shell.
+              This means that the shell's start-up files (<filename>.profile</filename> etc.) are read
+              and the current working directory is set to the target user's <envar>HOME</envar>.
+            </para>
+            <screen>
+<prompt>&exampleuser_plain;:~ &gt; </prompt>sudo -i
+root's password:
+<prompt>root:~ # </prompt>exit
+<prompt>&exampleuser_plain;:~ &gt; </prompt>
+            </screen>
+          </listitem>
+        </varlistentry>
+      </variablelist>
+    </sect2>
+
+    <sect2 xml:id="sec.adm.sudo.usage.env">
+      <title>Environment variables</title>
+      <para>By default, &sudo; does not propagate environment variables:</para>
+      <screen>
+&prompt.user;ENVVAR=test env | grep ENVVAR
+ENVVAR=test
+&prompt.user;ENVVAR=test sudo env | grep ENVVAR
+root's password:
+<co xml:id="co.sudo.env.novar"/>
+&prompt.user;
+      </screen>
+      <calloutlist>
+        <callout arearefs="co.sudo.env.novar">
+          <para>
+            The empty output shows that the environment variable <envar>ENVVAR</envar>
+            did not exist in the context of the command run with &sudo;.
+          </para>
+        </callout>
+      </calloutlist>
+
+      <para>This behaviour can be changed by the <literal>env_reset</literal> option, see <xref linkend="tab.adm.sudo.options"/>.</para>
+    </sect2>
+  </sect1>
+
+  <sect1 xml:id="sec.sudo.conf">
+    <title>Configuring &sudo;</title>
+
+    <para>
+      &sudo; is a very flexible tool with extensive configuration.
+    </para>
+
+    <note>
+      <title>Locked out of sudo</title>
+      <para>
+        Should you lock yourself out of &sudo; by mistake, you can use <command>su -</command>
+        and the &rootuser; password to get a root shell.
+      </para>
+    </note>
+
+    <sect2 xml:id="sec.sudo.conf.edit">
+      <title>Editing the configuration files</title>
+      <para>
+        The main policy configuration file for &sudo; is <filename>/etc/sudoers</filename>.
+        As it is possible to lock yourself out of the system if there are any errors in that file, it is
+        strongly recommended to use <command>visudo</command> for editing.
+        It will prevent simultaneous changes to the opened file and check for syntax errors before saving.
```

Comment:
It will lock the file to prevent simultaneous changes by others and perform a syntax check before saving.